src/device-status/config.py

```diff
@@ -17,18 +17,20 @@ class Config:
     TESTING = False
     CSRF_ENABLED = True
     DB_NAME = os.getenv("DB_NAME_PROD")
-    REGISTRY_MONGO_URI = os.getenv('REGISTRY_MONGO_GCE_URI')
-    MONITORING_MONGO_URI = os.getenv('MONITORING_MONGO_GCE_URI')
-    MAX_ONLINE_ACCEPTABLE_DURATION = int(os.getenv('MAX_ONLINE_ACCEPTABLE_DURATION', ONE_HOUR))
-    DUE_FOR_MAINTENANCE_DURATION = os.getenv('DUE_FOR_MAINTENANCE_DURATION', TWO_WEEKS)
+    REGISTRY_MONGO_URI = os.getenv("REGISTRY_MONGO_GCE_URI")
+    MONITORING_MONGO_URI = os.getenv("MONITORING_MONGO_GCE_URI")
+    MAX_ONLINE_ACCEPTABLE_DURATION = int(
+        os.getenv("MAX_ONLINE_ACCEPTABLE_DURATION", ONE_HOUR)
+    )
+    DUE_FOR_MAINTENANCE_DURATION = os.getenv("DUE_FOR_MAINTENANCE_DURATION", TWO_WEEKS)
     SECRET_KEY = os.getenv("SECRET_KEY")
-    BASE_API_URL = "https://staging-platform.airqo.net/api/v1/data"
+    BASE_API_URL = os.getenv("BASE_API_URL")
     RECENT_FEEDS_URL = f"{BASE_API_URL}/feeds/transform/recent"
 
 
 class ProductionConfig(Config):
     DEVELOPMENT = False
-    BASE_API_URL = "https://platform.airqo.net/api/v1/data"
+    BASE_API_URL = os.getenv("BASE_API_URL")
```

Comment:
[device status BASE_API_URL variable](https://github.com/airqo-platform/AirQo-api/pull/1120/commits/f942c99000dfc1df31f74ccb70824246a56cb1ed)

Reply:
Hi @123MwanjeMike 
What is the BASE_API_URL???
https://platform.airqo.net/api/v1/data or https://platform.airqo.net/api/v1/

Reply:
It's https://platform.airqo.net/api/v1/data @NoahNsimbe

Reply:
Hi @123MwanjeMike , could we also "mention" the ENVIRONMENTS in the names of these variables. Example:

- STAGING_BASE_API
- PRODUCTION_BASE_API

Unless this is not necessary?

Reply:
Hi @Baalmart 
That's not necessary given that we are using an environment variable ie on staging **BASE_API_URL**=https://staging-platform.airqo.net/api/v1/data, and **BASE_API_URL**=https://platform.airqo.net/api/v1/data on production.

Reply:
Thanks @123MwanjeMike , I still find it confusing for future maintainers but we move ahead! :)

Reply:
Specifying environment variables as staging/production creates unnecessary redundancy which I was trying to avoid.
Also, staging environment/secret values maybe shared with contributors with minimal risk as compared to production environment/secret values.